src/Chapter13/Listing13.15.ParameterlessStatementLambdas.cs

```diff
@@ -9,14 +10,15 @@ public static void Main()
             Func<string> getUserInput =
                 () =>
                 {
-                    string input;
+                    string? input;
                     do
                     {
                         input = Console.ReadLine();
                     }
-                    while(input.Trim().Length == 0);
-                    return input;
+                    while(!string.IsNullOrEmpty(input));
```

Comment:
The previous code was using `.Trim()` which would remove whitespace. So the equivalent here would be to check against null or whitespace.
```suggestion
                    while(!string.IsNullOrWhiteSpace(input));
```